### PR TITLE
Faq plus signs

### LINKED ISSUE #50: Add plus icon to FAQ's
- [ ] Received feedback from spencer to add in plus icons to indicate the faq's are clickable / expandable. 
Screen Shot 2022-11-02 at 5 11 38 PM

### PR DESCRIPTION
**Pull request by srazh**
Added plus signs to FAQ section

Fixes #50 

Changelist:

- FAQ section plus signs added
- not visible on mobile

Tested:

- localhost8000

Screenshots & Screencasts:

https://user-images.githubusercontent.com/63278213/200054236-74f17f2b-d1e3-4d78-a9eb-7081bbf7b6c3.mov

### CODE REVIEW

File: main-site/components/faq-section/faq-item/FaqItem.tsx [hunk 1]
```diff
@@ -23,7 +26,9 @@ const FaqItem: React.FC<FaqItemProps> = ({
           isSelected ? setSelectedQuestion(-1) : setSelectedQuestion(id)
         }
       >
+        {' '}
```
Comment (jzhzhang66) [inline main-site/components/faq-section/faq-item/FaqItem.tsx]: Is this space doing anything? If not let's remove!

File: main-site/components/faq-section/faq-item/FaqItem.styles.ts [hunk 2]
```diff
@@ -38,9 +39,20 @@ const StyledFaqDropdown = styled(P)`
   padding-left: 0;
 `;
 
+const StyledFaqPlusSign = styled.img`
+  height: 1.5em;
```
Comment (jzhzhang66) [inline main-site/components/faq-section/faq-item/FaqItem.styles.ts]: you can remove height and just have width and it'll work just fine!

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
Looks great!!

**Review by jzhzhang66 — APPROVED** [review]
LGTM KWEEEEEN

**Review by mike10911 — APPROVED** [review]
This looks so goooooodddd!!

**Pull request merged**